src/GraphBuilder.js

```diff
@@ -11,23 +12,80 @@ export default class GraphBuilder {
    */
   constructor(state) {
     this.state = state
+    this.dataset = rdf.dataset()
   }
 
   /**
    * @return {Graph} an RDF graph that represents the data in the state
    */
   get graph() {
-    const dataset = rdf.dataset()
+    const baseURI = rdf.namedNode('')
```

Comment:
Assuming that this is being set to `''` for now since URI not provided / minted. If so, should comment.

Reply:
This is just a relative URI.